ydr/ui.py

```diff
@@ -206,7 +217,8 @@ class SOLLUMZ_PT_LIGHT_TIME_FLAGS_PANEL(TimeFlagsPanel, bpy.types.Panel):
 
     @classmethod
     def poll(self, context):
-        return context.light is not None
+        obj = context.active_object
+        return obj is not None and obj.type == "LIGHT" and obj.sollum_type not in [LightType.NONE, SollumType.NONE]
```

Comment:
The last check should be:
```suggestion
        return obj is not None and obj.type == "LIGHT" and obj.sollum_type == SollumType.LIGHT
```
`LightType` is not used on `bpy.types.Object.sollum_type`, but on `bpy.types.Light.sollum_type`